src/components/LoaderRing/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -113,23 +113,23 @@ exports[`<LoaderRing /> renders LoaderRing correctly 1`] = `
 }
 
 <div
-  className="c0"
+  className="sc-AxjAm c0"
   display="block"
   size="regular"
 >
   <div />
   <div />
   <div />
   <div
-    className="c1"
+    className="sc-AxjAm c1"
     display="block"
     size="regular"
   >
     <div />
     <div />
     <div />
     <div
-      className="c2"
+      className="sc-AxjAm c2"
```

Comment:
Please investigate why it outputs generated classnames with jest-styled-components serializer

Reply:
jest-styled-components issue